src/backend/_scss/application.scss

```diff
@@ -2097,3 +2097,7 @@ a.skip:hover {
   margin-bottom: 5px;
   margin-left: 5px;
 }
+
+.center{
+  display: block;
```

Comment:
@aayusharora I didn't write the class name according to its properties but just the functionality it is serving , should it be changed to display or something else please mention?